src/base/bittorrent/session.cpp

```diff
@@ -4764,10 +4764,10 @@ void Session::handleSessionStatsAlert(const lt::session_stats_alert *p)
     m_status.diskWriteQueue = stats[m_metricIndices.peer.numPeersDownDisk];
     m_status.peersCount = stats[m_metricIndices.peer.numPeersConnected];
 
-    const int numBlocksRead = stats[m_metricIndices.disk.numBlocksRead];
-    const int numBlocksCacheHits = stats[m_metricIndices.disk.numBlocksCacheHits];
+    const long numBlocksRead = stats[m_metricIndices.disk.numBlocksRead];
+    const long numBlocksCacheHits = stats[m_metricIndices.disk.numBlocksCacheHits];
     m_cacheStatus.totalUsedBuffers = stats[m_metricIndices.disk.diskBlocksInUse];
-    m_cacheStatus.readRatio = static_cast<qreal>(numBlocksCacheHits) / std::max(numBlocksCacheHits + numBlocksRead, 1);
+    m_cacheStatus.readRatio = static_cast<qreal>(numBlocksCacheHits) / std::max(numBlocksCacheHits + numBlocksRead, 1L);
```

Comment:
Why?

Reply:
Required as part of turning `numBlocksCacheHits` and `numBlocksRead` into `auto` or `long`.

Reply:
I think this is redundant, C++ will coerce (automatically) to the proper type.

Reply:
My static analyzer was throwing an error saying there was no `std::max` candidate for int and long, but I will try actually building and see if there truly is an error or not.

Reply:
I can't compile _with_ this change.

Reply:
I fixed the problem, it was supposed to be `1LL`, not `1L` anyway.

Reply:
IMO this should still be reverted.

Reply:
Alright, fixed.